CSharpPatterns/CSharpPatterns/IDeveloper.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSharpPatterns
+{
+    public interface IDeveloper //Interface for a developer
```

Comment:
what the reason to create this interface?

Reply:
this interface declares Create() method, which should return IVehicle objects